calicoctl.py

```diff
@@ -228,6 +235,10 @@ def node(ip, node_image):
     # Set up etcd
     client = DatastoreClient()
 
+    # Enable IP forwarding.
```

Comment:
Might be beneficial to future generations to describe what we're enabling forwarding for.